src/gromacs/nblib/topology.h

```diff
@@ -68,6 +68,24 @@ std::vector<gmx::ExclusionBlock> toGmxExclusionBlock(const std::vector<std::tupl
 // Add offset to all indices in inBlock
 std::vector<gmx::ExclusionBlock> offsetGmxBlock(std::vector<gmx::ExclusionBlock> inBlock, int offset);
 
+//! Helper class for Topology to keep track of particle IDs
+class EnumerationKey
```

Comment:
I guess this is implicitly tested by TopologyHasSequencing, but would more explicit testing be warranted/possible?

Reply:
There's no logic inside Topology to serve calls accessing the EnumerationKey object, so I see two options: either having the sequencing test on the Topology object as it is now, or move the test to EnumerationKey only. I don't really have a strong preference for either approach.